crates/fluvio-spu/src/replication/leader/leaders_state.rs

```diff
@@ -1,19 +1,14 @@
-use std::{
-    ops::{Deref},
-};
+use std::ops::Deref;
```

Comment:
Same comment as previously. let's move formatting changes out of this PR

Reply:
If I modify a file where there are warnings from the rust-analyzer, I fix them. In that way, such warnings will be gradually fixed. Nobody ever will fix those warnings separately unless all tasks on the project are gone.